src/sentry/static/sentry/app/views/settings/projectPlugins/details.tsx

```diff
@@ -13,9 +15,21 @@ import ExternalLink from 'app/components/links/externalLink';
 import PluginConfig from 'app/components/pluginConfig';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import withPlugins from 'app/utils/withPlugins';
-import withOrganization from 'app/utils/withOrganization';
-import withProject from 'app/utils/withProject';
 import {trackIntegrationEvent} from 'app/utils/integrationUtil';
+import space from 'app/styles/space';
+import {Plugin, Organization, Project} from 'app/types';
+
+type Props = {
+  organization: Organization;
+  project: Project;
+  plugins: {
+    plugins: Plugin[];
+  };
+} & WithRouterProps<{orgId: string; projectId: string; pluginId: string}>;
```

Comment:
maybe you can replace it with `RouteComponentProps<{orgId: string; projectId: string; pluginId: string}, {}>` ?

Reply:
@priscilawebdev I could but is that any better? We need that extra argument of `{}` which is annoying IMO.

Reply:
Hm... I am not sure, but I noticed that we usually use RouteComponentProps